### PR TITLE
Password api

### PR DESCRIPTION
**Pull request by aftab76ahmad76**
Password change api

### CODE REVIEW

File: app/tests/test_user.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from rest_framework import status
+
+from app.tests.base import BaseTestCase
+
+
+class UserTestCase(BaseTestCase):
+    def test_user_change_password(self):
```
Comment (getalihassan) [inline app/tests/test_user.py:7]: this is only positive test case, can you write negative test cases as well?

**Review by getalihassan — CHANGES_REQUESTED** [review]

File: app/serializers.py [hunk 2]
```diff
@@ -131,3 +131,20 @@ class Meta:
...
+
+
+class UserPasswordSerializer(serializers.Serializer):
+    password = serializers.CharField(min_length=8, required=True)
+    old_password = serializers.CharField()
+
+    def validate(self, data):
+        old_password = data["old_password"]
+        password = data["password"]
+        if not (
```
Comment (getalihassan) [inline app/serializers.py:143]: raise two different messages depending on the check

Comment (getalihassan) [reply app/serializers.py:143]: Password does not match
New password cannot be same as the old password.

**Review by getalihassan — CHANGES_REQUESTED** [review]

File: app/tests/test_user.py (hunk 1 above)
Comment (aftab76ahmad76) [reply app/tests/test_user.py:7]: added

File: app/serializers.py (hunk 2 above)
Comment (aftab76ahmad76) [reply app/serializers.py]: done

**Review by getalihassan — APPROVED** [review]

**Pull request merged**